boards/arm/sam4e_xpro/doc/index.rst

```diff
@@ -0,0 +1,163 @@
+
+.. image:: img/sam4e_xpro.jpg
+     :width: 500px
+     :align: center
+     :alt: SAM4E Xplained Pro
+
+Hardware
+********
+
+- ATSAM4E16E ARM Cortex-M4F Processor
+- 12 MHz crystal oscillator
+- internal 32.768 kHz crystal oscillator
+- 2 x IS61WV5128BLL 4Mb SRAM
+- MT29F2G08ABAEAWP 2Gb NAND
+- SD card connector
+- CAN-bus (TLE7250GVIOXUMA1 CAN Transceiver)
+- Ethernet port (KSZ8081MNXIA phy)
+- Micro-AB USB device
+- Micro-AB USB debug interface supporting CMSIS-DAP, Virtual COM Port and Data
+  Gateway Interface (DGI)
+- One reset and one user pushbutton
+- 1 yellow user LEDs
+
+
+Supported Features
+==================
+
+The sam4e_xpro board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the Kconfig
+:zephyr_file:`boards/arm/sam4e_xpro/sam4e_xpro_defconfig`.
+
+Connections and IOs
+===================
+
+The `SAM4E Xplained Pro User Guide`_ has detailed information about board
+connections. Download the `SAM4E Xplained Pro documentation`_ for more detail.
+
+System Clock
+============
+
+The SAM4E MCU is configured to use the 12 MHz internal oscillator on the board
+with the on-chip PLL to generate an 120 MHz system clock.
+
+Serial Port
+===========
+
+The ATSAM4E16E MCU has 2 UARTs and 2 USARTs. One of the UARTs (UART0) is
+configured for the console and is available as a Virtual COM Port by EDBG USB
+chip.
+
+Programming and Debugging
+*************************
+
+Flashing the Zephyr project onto SAM4E MCU requires the `OpenOCD tool`_.
+Support for Atmel SAM4E microcontroller series was added in OpenOCD release
+0.10.0, which was added in Zephyr SDK 0.9.2.
```

Comment:
This comment was added back at the time when there was no restriction on Zephyr SDK version and OpenOCD 0.10.0 was not available on the majority of the distributions. Since currently we're forcing users to always use the latest Zephyr SDK and OpenOCD 0.10.0 is widely available I propose to drop the sentence.